server/hotrod/pom.xml

```diff
@@ -199,6 +208,50 @@
                </instructions>
             </configuration>
          </plugin>
+
+         <plugin>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-protocol-parser-generator-maven-plugin</artifactId>
+            <version>${project.version}</version>
+            <configuration>
+               <files>
+                  <file>${basedir}/src/main/resources/hotrod.gr</file>
+               </files>
+            </configuration>
+            <executions>
+               <execution>
+                  <goals>
+                     <goal>generate</goal>
```

Comment:
I wonder if we can specify a phase here so that the plugin isn't needed for `mvn clean`. I usually run the build through a script that does `mvn clean` first, and it's annoying when it stops working because I don't have the plugin in my local repo (e.g. after a version change)

Reply:
This worked:

               <execution>
                  <goals>
                     <goal>generate</goal>
                  </goals>
                  <phase>generate-sources</phase>
               </execution>